jsapp/js/components/header/organizationBadge.module.scss

```diff
@@ -1,12 +1,20 @@
 @use 'scss/colors';
 
 .root {
-  color: colors.$kobo-white;
-  background-color: colors.$kobo-gray-700;
   padding: 6px 10px;
   border-radius: 48px;
   font-weight: 600;
   font-size: .85em;
-  line-height: 12px;
   margin-right: 20px;
+  width: fit-content;
+}
+
+.header {
```

Comment:
Since the only thing we are changing based on where this component is being used is color, let's just have the component take 'badgeColor' as a prop. That way if someone ends up using a light blue org badge elsewhere in the app they won't need to make a new class for it.